### PR TITLE
Add demotivational command in fun category.

### PR DESCRIPTION
**Pull request by SharifPoetra**
With this pull request, Its also pushing the master to dev branch,  Just wondering why the master always up-to-date than dev? @KurokuTetsuya

### CODE REVIEW

**Comment by KurokuTetsuya** [comment]
> With this pull request, Its also pushing the master to dev branch, Just wondering why the master always up-to-date than dev? @KurokuTetsuya

Lah mana saya tau, siapa juga yang nge update master dluan? Gua pushnya ke dev anjer, kasian itu travis ci di commit terus, ntar ngebuild lagi..

**Comment by SharifPoetra** [comment]
Liat github logs lah siapa yg push ke master deluan sebelum dev

File: .gitignore [hunk 1]
```diff
@@ -5,10 +5,13 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+<<<<<<< HEAD
+=======
```
Comment (dwicao) [inline .gitignore]: remove this conflict message

File: .gitignore [hunk 2]
```diff
@@ -5,10 +5,13 @@ npm-debug.log*
...
 # Lockfiles
 package-lock.json
 yarn.lock
 
+>>>>>>> b608d627540287c262c7444a87b1445d2774edd8
```
Comment (dwicao) [inline .gitignore]: same here

**Comment by KurokuTetsuya** [comment]
Ahh, bukannya gua push ke dev then gua merge?

File: .gitignore (hunk 2 above)
Comment (KurokuTetsuya) [reply .gitignore]: Okay then

File: .gitignore (hunk 1 above)
Comment (KurokuTetsuya) [reply .gitignore]: 😄

File: src/commands/fun/DemotivationalCommand.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		if (flags[2] && flags[2] === "avatar") {
+			user = message.mentions.users.first();
+		} else {
+			return message.channel.send("Please mention someone to use their avatar as the poster using the flags `--avatar`!");
+		}
+		
+		const { body } = thread.sync(get("https://emilia-api.glitch.me/api/demotivational").query({
+			title,
+			text,
+			image: user.displayAvatarURL({ format: "png", size: 2048 })
```
Comment (dwicao) [inline src/commands/fun/DemotivationalCommand.js]: I think it's better to just give it `1024` size

Comment (dwicao) [reply src/commands/fun/DemotivationalCommand.js]: *better (in term of performance)

**Comment by SharifPoetra** [comment]
Alright thanks!

**Pull request merged**